.eslintrc.yaml

```diff
@@ -17,8 +20,11 @@ settings:
   wc:
     elementBaseClasses: ["BaseElement", "LitElement", "FormElement"]
 rules:
+  # Rules disabled in favor of clang-format
+  "@typescript-eslint/indent": off
```

Comment:
Yes, I left it because it would be nice to solve it.
But now I disable it! https://github.com/material-components/material-components-web-components/pull/371/commits/35330ff72eef467b75242c509a230a34aff62455